_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+talk describes a novel way of combining extension blocks and the
+time-warp attack to allow both upgraded nodes and old nodes to gain the
+same capacity increase and see all the same transactions for validation
+while simultaneously slightly reducing mining centralization risk.
+Upgraded nodes would validate one or more extension blocks providing
+additional block space with a 15 minute average interval, but they'd
+also restrict the time stamps in legacy blocks to ensure a permanent
+(but limited) time warp attack increased the frequency of legacy blocks
+enough to allow them to include the same transactions that previously
+appeared in the extension blocks.
+
+*[video][vid forward blocks], [transcript][tx forward blocks],
+[paper][paper forward blocks], soft fork*
+
+### Compact multi-signatures for smaller blockchains
+
+*Research by Dan Boneh, Manu Drijvers and Gregory Neven*
+
+This talk describes an alternative to the Bellare-Neven-based Schnorr
+signature scheme described in the [MuSig paper][] and [proposed as a
+standard][schnorr pre-bip] for Bitcoin by Pieter Wuille.  The
+alternative method makes use of [pairing-based cryptography][],
+specifically an adaptation of the [Boneh-Lynn-Shacham (BLS) signature
+scheme][bls sigs].  Although pairing-based schemes require an additional
+fundamental security assumption beyond those made by both Bitcoin's
+current ECDSA scheme and proposed Schnorr scheme, the authors present
+evidence that their scheme would produce smaller signatures in general,
+be much more computationally efficient in some multisignature use cases,
+and make it possible to prove which members of the set of signers
+actually worked together to create a threshold signature (i.e.  k-of-m
+signers, e.g. 2-of-3 multisig).
+
+*[video][vid bls msig], [transcript][tx bls msig], [paper
+(pre-print)][paper bls msig], soft fork using segwit script upgrade
+mechanism*
+
+### Accumulators: a scalable drop-in replacement for merkle trees
+
+*Research by Benedikt Bünz, Benjamin Fisch and Dan Boneh*
+
+In Bitcoin and other cryptocurrencies, scalable commitments to sets of
+information---such as transactions or UTXOs---are normally made using
+merkle trees that allow proving an element is a member of the set by
+generating a proof whose size and validation cost is roughly *log2(n)*
+for a set of *n* elements.
+
+This talk describes an alternative method based on RSA accumulators that
+provides many potential benefits: the size of a proof is constant no
+matter how many elements are members of the set, adding or removing
+elements from an accumulator can be efficiently batched (e.g. one update
+per block), and validation that one or more elements are members of a
+set could potentially be much faster (although this has not yet been
+fully tested).
+
+*[video][vid accumulators], [transcript][tx accumulators], soft fork for
+new applications such as UTXO set commitments*
+
+### Scriptless ECDSA
```

Comment:
The name of the talk was "Instantiating [Scriptless] 2P-ECDSA: Fungible 2-of-2 MultiSigs for Today's Bitcoin". I think a better heading for this section would be "Implementing two party ECDSA" or similar.

Reply:
I think "two-party ECDSA" is an unfamiliar and non-descriptive term (isn't all ECDSA two-party, as in why could you create keys and sigs if it was just you? ;-)  So I'm changing the title just slightly; however, feel free to change it to whatever you think is most approprate before merge.

Reply:
I think the new text for this section is excellent, but I think 'Scriptless ECDSA security for Lightning Network payment channels' still isn't quite right (isn't all ECDSA scriptless? 🙂). I'm going to change the section title to "Multiparty ECDSA for scriptless Lightning Network payment channels". I hope that's ok for you!